tools/ansible/roles/dockerfile/templates/supervisor.conf.j2

```diff
@@ -75,8 +74,8 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
-[program:awx-rsyslogd]
-command = rsyslogd -n -i /var/run/awx-rsyslog/rsyslog.pid -f /var/lib/awx/rsyslog/rsyslog.conf
+[program:awx-cache-clear]
+command = awx-manage run_cache_clear
```

Comment:
if kube_dev conditional need to exist look at how conditional around uwsgi is done

Reply:
@djyasin can u address this comment

Reply:
Resolved via this pr: https://github.com/ansible/awx/pull/13771